doc/source/user_guide/groupby.rst

```diff
@@ -1187,12 +1187,6 @@ is only interesting over one column (here ``colname``), it may be filtered
    If you do wish to include decimal or object columns in an aggregation with
    other non-nuisance data types, you must do so explicitly.
 
-.. warning::
```

Comment:
should this be removed now or wait until a deprecation is enforced in the groupby code?

Reply:
Thanks - got confused on where this was.